.github/workflows/build_and_test.yml

```diff
@@ -14,9 +14,9 @@ jobs:
     runs-on: macOS-latest
     env:
         # The XCode version to use. If you want to update it please refer to this document:
-        # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners#xcode
+        # https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners#supported-software
         # and set proper version.
-        XCODE_VERSION: 11.5
+        XCODE_VERSION: 12.2
```

Comment:
Use `12.1` instead which is now available on GH Actions

Reply:
isn't it going to fail due to the missing corenfc framework?

Reply:
The CoreNFC bug is fixed in the 12.1 GM :) At least I haven't had a problem with it locally